src/pages/event/eventDetail/eventDetail.tsx

```diff
@@ -0,0 +1,179 @@
+import React, { FunctionComponent } from "react";
+import { useLocation, useParams } from "react-router-dom";
+import { EventProps } from "../../../components/UI/ResourceEvent";
+import { events } from "../event";
+import { Helmet } from "react-helmet";
+import { Link } from "react-router-dom";
+import { ChevronLeft, ExternalLink, PlayCircle, Download } from "react-feather";
+import { Page } from "../../../components/Layout/Page";
+import { isFuture, format } from "date-fns";
+import { formatTime } from "../../../common/utils/dateTime";
+import { LinkButton } from "@govtechsg/tradetrust-ui-components";
+import ReactMarkdown from "react-markdown";
+
+export const EventPageDetail: FunctionComponent = () => {
+  const locationPath = useLocation();
+  const params: { slug: string } = useParams();
+  const detail: EventProps = events.find((eventDetail) => eventDetail.slug === params.slug);
+
+  const {
+    title,
+    thumbnail,
+    blurb,
+    link,
+    date,
+    timeStart,
+    timeEnd,
+    videoLink,
+    slides,
+    registerLink,
+    location,
+    downloadableMediaContent1,
+    downloadableMediaContent2,
+    downloadableMediaContent3,
+    eventDetails,
+  } = detail.attributes;
+  const downloadableContents = [downloadableMediaContent1, downloadableMediaContent2, downloadableMediaContent3];
+
```

Comment:
use list -> https://www.netlifycms.org/docs/widgets/list/#list

Reply:
cos it should be a downloadable content, list cant do it like using widget file. and i cant enable multiple file upload. so strange.

Reply:
ok updated to use list